azure-pipelines.yml

```diff
@@ -13,6 +15,44 @@ steps:
 - script: pip install -U iotedgedev
   displayName: 'Install IoT Edge Dev Tool'
 
+- task: DotNetCoreCLI@2
+  displayName: 'Build C# projects and run unit tests'
+  inputs:
+    command: 'test' 
+    projects: '**/*.csproj'
+    arguments: '--collect:"XPlat Code Coverage" -- RunConfiguration.DisableAppDomain=true'
+
+- task: DotNetCoreCLI@2
+  displayName: "Install dotnet format"
+  inputs:
+    command: 'custom'
+    custom: 'tool'
+    arguments: 'install -g dotnet-format'
+
+- script: |
+   dotnet format tests/TimeSeriesRepository.Test/ --check --fix-style warn --fix-whitespace 
```

Comment:
Besides the unit test project and the `RedisTimeSeriesEdge` module, all other modules contain non-C# code, hence not sure if it brings value to run `dotnet format` on them.

Reply:
I was thinking to have it in case we add to the repo in the future, but makes sense to remove them and re add them when necessary.